healpy/src/_pixelfunc.pyx

```diff
@@ -41,7 +41,7 @@ def ringinfo(nside, np.ndarray[int64, ndim=1] ring not None):
            -0.33333333, -0.66666667]), array([ 0.        ,  0.39965263,  0.74535599,  0.94280904,  1.        ,
             0.94280904,  0.74535599]), array([ True,  True,  True, False,  True, False,  True], dtype=bool))
     """
-    if not isnsideok(nside):
+    if not isnsideok(nside, nest=True):
```

Comment:
This whole function should be done on RING-ordered Healpix bases, so that it works for all Nsides.